server/src/dash-server.ts

```diff
@@ -84,6 +89,31 @@ export class DashServer {
     res.end();
   }
 
+  async addPushSubscription(req: express.Request, res: express.Response) {
+    if (!req.body) {
+      res.sendStatus(400);
+      return;
+    }
+
+    const token = req.cookies['id'];
+    const loginResult = await this.github.query<ViewerLoginQuery>(
```

Comment:
We should generally avoid this at some point. When we first exchange for a token, we should store the user login with the token somewhere. Maybe just add a TODO here about not having to do this query again?